.gitignore

```diff
@@ -101,3 +101,4 @@ gmailLogin.properties
 webLogin.properties
 mailHandlerTestProps.properties
 config.yaml
+mturk.properties
```

Comment:
Do we need those separate `.properties`? Can't we just merge them into the main config?